### ISSUE TITLE
Request to add a secondary language

### ISSUE THREAD

**Issue by jojorne** [itsecurityco/to-google-translate#125]
The ability to add other languages to the context menu. Sometimes english is not your mother tongue, even tho you use english a lot, there are cases where you don't know the meaning in english.

**Comment by marxenegls** [comment]
exactly - you have no way of setting or even choosing a second, third language , overlay: autodetect / select  would be something i'd settle for

**Comment by farindk** [comment]
Yes, the ability to add several target languages would also be very useful to me. Especially since I often have to translate a text _to_ a third language.

**Comment by Yabla1** [comment]
Absolutely, maybe make this feature as tabs, so you can switch between languages easily -- just like the original page: translate.google.com.
